pkg/gui/controllers/helpers/textbox_helper.go

```diff
@@ -0,0 +1,151 @@
+package helpers
```

Comment:
I copy and paste lots from `confirmation_helper.go`
It has many functionalities about input, menu, and commit_message.
Tell me freely some ideas on how to organize functions in confirmation_helpers.go and textbox_helpers.go. 🙇